src/app/component/ProductReviews/ProductReviews.container.js

```diff
@@ -67,6 +70,32 @@ export class ProductReviewsContainer extends PureComponent {
         showPopup({ title: __('Write a new review') });
     }
 
+    scrollToPosition(ref, isContentExpanded) {
+        const elem = ref && ref.current;
+
+        if (elem) {
+            setTimeout(() => {
```

Comment:
Is `setTimeout` really necessary here?
Maybe `requestAnimationFrame` is going to suit you better?